packages/vscode-dependencies-validation/.eslintignore

```diff
@@ -0,0 +1,6 @@
+node_modules
```

Comment:
please evaluate option for extacting common eslint ignore to settings to root of mono repo